### ISSUE TITLE
InlineCssPostProcessor overwrites default tag styles 

### ISSUE THREAD

**Issue by TwentyFourMinutes** [SebastianStehle/mjml-net#203]
Since `v4` the new `InlineCssPostProcessor` causes the default styles defined by mjml-net to be overwritten. I believe its caused on this line: 
https://github.com/SebastianStehle/mjml-net/blob/4aa0c64f46df824c0e96fc19232c690e9cdc780a/Mjml.Net.PostProcessors/InlineCssPostProcessor.cs#L37

From what I can tell this more or less ignores all styles defined on the `style=` attribute. For example calling `element.GetAttribute(TagNames.Style)` will correctly return me `word-break:break-word`. Otherwise with the current implementation the whole `InlineCssPostProcessor` causes the whole HTML to be more or less broken.

Would be awesome if you could take a look at it :). Also thanks for maintaining this library, it's awesome!

**Comment by SebastianStehle** [comment]
Hi,

what do you mean with "default" styles?

**Comment by TwentyFourMinutes** [comment]
Things such as these https://github.com/SebastianStehle/mjml-net/blob/4aa0c64f46df824c0e96fc19232c690e9cdc780a/Mjml.Net/Components/Body/ColumnComponent.cs#L223

I came to this conclusion, as texts in my email which are longer than 600px now overflow way outside their container. I then noticed that for example `word-break: break-word` is no longer present in the html at all when upgrading to **v4**.

**Comment by SebastianStehle** [comment]
Good point, I think it is easy to fix. Just don#t overwrite styles that already exist I guess (except they are important)

**Comment by TwentyFourMinutes** [comment]
@SebastianStehle I already gave it a shot with:
````cs
private static void InlineStyle(IElement element, IStyleCollection styleCollection)
{
    var currentStyle = styleCollection.GetDeclarations(element);

    if (!currentStyle.Any())
    {
        return;
    }

    var style = element.GetStyle();

    foreach (var property in currentStyle)
    {
        style.SetProperty(property.Name, property.Value, property.IsImportant ? CssKeywords.Important : null);
    }
````

However, I still had the issue that `element.GetStyle()` won't contain the `word-break: break-word` property 🤔

**Comment by SebastianStehle** [comment]
Weird, perhaps there is another method that helps?

**Comment by SebastianStehle** [comment]
I am not sure if I understand the bug tbh. I have this test:

```
<mjml-test>
  <mj-head>
    <mj-style inline=""inline"">
      .red-text div {
        color: red !important;
        font-size: 1.5rem;
        font-weight: light;
      }
    </mj-style>
  </mj-head>
  <mj-body>
    <mj-raw>
      <div class=""red-text"">
        <div style=""font-weight: bold""></div>
      </div>
    </mj-raw>
  </mj-body>
</mjml-test>
```
Here font-weight is not overriden. Because GetDeclarations actually calculates the styles in the correct logic. How would that override styles from mjml?

**Comment by TwentyFourMinutes** [comment]
@SebastianStehle look at this modified variant of the the test

```cs
    [Fact]
    public async Task Should_render_inline()
    {
        var source = @"
<mjml-test>
  <mj-head>
    <mj-style inline=""inline"">
      .red-text div {
        color: red !important;
        font-size: 1.5rem;
      }
    </mj-style>
  </mj-head>
  <mj-body>
    <mj-raw>
      <div class=""red-text"">
        <div style=""font-weight: bold; word-break: break-word;""></div>
      </div>
    </mj-raw>
  </mj-body>
</mjml-test>
";

        var (result, _) = await TestHelper.RenderAsync(source, new MjmlOptions
        {
            PostProcessors =
            [
                AngleSharpPostProcessor.Default
            ],
            Beautify = true
        }, helpers: [new StyleHelper()]);

        AssertHelpers.HtmlFileAssert("Components.Outputs.StyleInline.html", result);
    }
```

I just added the ` word-break: break-word;` style, which I guess simulates what this does: https://github.com/SebastianStehle/mjml-net/blob/4aa0c64f46df824c0e96fc19232c690e9cdc780a/Mjml.Net/Components/Body/ColumnComponent.cs#L223 

And the test is still green! I can also verify that `word-break: break-word; is no longer present in the `result.Html`.

**Comment by SebastianStehle** [comment]
For me it looks like it just ignores most CSS properties.

**Comment by SebastianStehle** [comment]
I have tried to debug it in AngleSharp and created a bug report there. Lets see what happens

**Comment by TwentyFourMinutes** [comment]
@SebastianStehle Thank you very much. :)

**Comment by SebastianStehle** [comment]
Has been fixed
